libp2p/NodeTable.cpp

```diff
@@ -528,16 +530,17 @@ void NodeTable::onPacketReceived(
                 auto& in = dynamic_cast<PingNode&>(*packet);
                 in.source.setAddress(_from.address());
                 in.source.setUdpPort(_from.port());
-                addNode(Node(in.sourceid, in.source));
-                
+                auto entry = addNode(Node(in.sourceid, in.source));
+                if (entry)
+                    entry->lastPongSentTime = RLPXDatagramFace::secondsSinceEpoch();
+
+                // Send PONG response.
```

Comment:
We shouldn't send PONG if it failed to be added

Reply:
Changed.